include/aws/mqtt/client.h

```diff
@@ -422,6 +443,23 @@ int aws_mqtt_client_connection_set_reconnect_timeout(
     uint64_t min_timeout,
     uint64_t max_timeout);
 
+/**
+ * Sets the callbacks to call when a connection succeeds or fails
+ *
+ * \param[in] connection                The connection object
```

Comment:
Where is this doc comment markup coming from?  This isn't what we use everywhere else/

Reply:
Ohh, sorry I didnt see the comment until I merged it. 
That's what we have been using in 311. For examples:
https://github.com/awslabs/aws-c-mqtt/blob/59a1e891a9be856e389f2efa79b6426d2c2971d8/include/aws/mqtt/client.h#L141